pkg/generators/models/testdata/cases/validation/generated/model_person.go

```diff
@@ -0,0 +1,218 @@
+// This file is auto-generated, DO NOT EDIT.
+//
+// Source:
+//     Title: Test
+//     Version: 0.1.0
+package generatortest
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+
+	"time"
+)
+
+// Person is an object.
+type Person struct {
+	// Age:
+	Age float32 `json:"age,omitempty"`
+	// Base64:
+	Base64 string `json:"base64,omitempty"`
+	// Date:
+	Date string `json:"date,omitempty"`
+	// Datetime:
+	Datetime time.Time `json:"datetime,omitempty"`
+	// Email:
+	Email string `json:"email,omitempty"`
+	// Gender:
+	Gender Gender `json:"gender"`
+	// Hostname:
+	Hostname string `json:"hostname,omitempty"`
+	// Ip:
+	Ip string `json:"ip,omitempty"`
+	// Ipv4:
+	Ipv4 string `json:"ipv4,omitempty"`
+	// Ipv6:
+	Ipv6 string `json:"ipv6,omitempty"`
+	// Name:
+	Name string `json:"name"`
+	// Url:
+	Url string `json:"url,omitempty"`
+	// Uuid:
+	Uuid string `json:"uuid,omitempty"`
+}
+
+// Validate implements basic validation for this model
+func (m Person) Validate() error {
+	return validation.Errors{
+		"age": validation.Validate(
+			m.Age, validation.Min(float32(18)), validation.Max(float32(120)),
+		),
+		"base64": validation.Validate(
+			m.Base64, is.Base64,
+		),
+		"date": validation.Validate(
+			m.Date, validation.Date("2006-01-02"),
+		),
+		"datetime": validation.Validate(
+			m.Datetime, validation.Date(time.RFC3339),
+		),
+		"email": validation.Validate(
+			m.Email, is.Email,
```

Comment:
This should probably use `is.EmailFormat` instead

https://github.com/go-ozzo/ozzo-validation/blob/34bd5476bd5bb4884aee8252974da4cd4e878a75/is/rules.go#L128-L131

```go
	// Email validates if a string is an email or not. It also checks if the MX record exists for the email domain.
	Email = validation.NewStringRuleWithError(govalidator.IsExistingEmail, ErrEmail)
	// EmailFormat validates if a string is an email or not. Note that it does NOT check if the MX record exists or not.
	EmailFormat = validation.NewStringRuleWithError(govalidator.IsEmail, ErrEmail)
```

Reply:
done